tool/tctl/common/admin_action_test.go

```diff
@@ -384,6 +387,14 @@ func (s *adminActionTestSuite) testTokens(t *testing.T) {
 		return s.authServer.DeleteToken(ctx, token.GetName())
 	}
 
+	createTokens := func() error {
+		return trace.NewAggregate(s.authServer.CreateToken(ctx, token), s.authServer.CreateToken(ctx, token2))
```

Comment:
Split lines to the 2 operations are more apparent?

```suggestion
		return trace.NewAggregate(
			s.authServer.CreateToken(ctx, token),
			s.authServer.CreateToken(ctx, token2),
		)
```